endpoints/openapi_generator.py

```diff
@@ -839,7 +823,8 @@ def __api_openapi_descriptor(self, services, hostname=None):
     for service in services:
       remote_methods = service.all_remote_methods()
 
-      for protorpc_meth_name, protorpc_meth_info in remote_methods.iteritems():
+      for protorpc_meth_name in sorted(remote_methods.iterkeys()):
```

Comment:
What is the purpose of this change?

Reply:
The order of iteration over Python dictionaries is undefined; I wanted to have a stable ordering.